gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineIdea.groovy

```diff
@@ -311,4 +329,31 @@ class BaselineIdea extends AbstractBaselinePlugin {
             module.dependencies.addAll(projectRefs)
         }
     }
+
+    /**
+     * Configures some defaults on the save-actions plugin, but only if it hasn't been configured before.
+     */
+    private static void configureSaveActions(Node rootNode) {
+        GroovyXmlUtils.matchOrCreateChild(rootNode, 'component', [name: 'SaveActionSettings'], [:]) {
+            // Configure defaults if this plugin is configured for the first time only
+            appendNode('option', [name: 'actions']).appendNode('set').with {
+                appendNode('option', [value: 'activate'])
+                appendNode('option', [value: 'noActionIfCompileErrors'])
+                appendNode('option', [value: 'organizeImports'])
+                appendNode('option', [value: 'reformat'])
+            }
+            appendNode('option', [name: 'configurationPath', value: ''])
+            appendNode('inclusions').appendNode('set').with {
+                appendNode('option', [value: 'src/.*\\.java'])
```

Comment:
nice

Reply:
Yep, so it's a bit tricky as if people already use this plugin (and have configured it for a project) then we do nothing.. but in the majority of cases, this is probably what people want.

An unsolved problem I will add to the description is how to deal with changing flags / how do we know what's safe to update because we set it up that way vs the user set it up that way.

Reply:
I'd have to look into what other options the plugin exposes, but we may want to narrow what users can configure to a very narrow set of flags